tests/wandb_settings_test.py

```diff
@@ -995,3 +995,13 @@ def test_local_api_key_validation():
 
     # ensure that base_url is applied first without causing an error in api_key validation
     wandb.Settings()._apply_settings(s)
+
+
+def test_run_urls():
```

Comment:
This is a "replacement" test for the one in `wandb_run_test`.

Reply:
do you mean `test_run_urls`?